src/core/operationRepo/OperationRepo.ts

```diff
@@ -16,6 +18,12 @@ export class OperationRepo {
         this._processDelta(delta);
       },
     );
+
+    setInterval(() => {
+      if (this._deltaQueue.length > 0) {
```

Comment:
Length check seems unnecessary, can we remove it?

Reply:
The check seems necessary:
Removing the check fails the following in [`executor.test.ts`](https://github.com/OneSignal/OneSignal-Website-SDK/blob/main/__test__/unit/core/executors.test.ts):
```
✕ Subscriptions executor flushes deltas at end of `processDeltaQueue` (17 ms)
✕ Identity executor flushes deltas at end of `processDeltaQueue` (10 ms)
✕ User Properties executor flushes deltas at end of `processDeltaQueue` (9 ms)
```
Where in each `processDeltaQueueSpy` is called 2 times instead of 1

Reply:
After investigating, it looks like the tests were failing due to using to using the incorrect jest timer. More details in my commit here: https://github.com/OneSignal/OneSignal-Website-SDK/pull/1179/commits/f680239c6719e06c49f9dd5b83b7c63206851174

I think the length check is unnecessary and that we should remove it